spec/unit/resources/hab_build_spec.rb

```diff
@@ -3,42 +3,24 @@
 
 describe 'test::build' do
   describe ':build action' do
-    let(:cleanup) { false }
-
     let(:chef_run) do
```

Comment:
FWIW, this `let` used to be `cached` before it needed to take the `cleanup` parameter.